.gitignore

```diff
@@ -4,7 +4,7 @@
 target
 *.iml
 application-local.yml
-
+localhost.p12
```

Comment:
had to add the p12 file to the server directory, added it to gitignore.... not sure if that was the correct thing to do.